### PR TITLE
Landing improvements

### PR DESCRIPTION
**Pull request by dtuite**

### CODE REVIEW

File: src/components/home/features/ServiceCatalog.js
Comment (iain-b) [inline src/components/home/features/ServiceCatalog.js:19]: I'm not sure about the line above here. I think it's the "no idea" part. I dunno I reckon customers would like to think they've some idea 😄 Nit pick so feel free to ignore; I don't have a better suggestion.

File: src/pages/index.js
Comment (iain-b) [inline src/pages/index.js:56]: Just an idea maybe "invert" the suggestion suggest that they can put their tribal knowledge into backstage and _speed up_ development. On the other hand I suppose you're trying to get them to think of the problem we're solving...

**Review by iain-b — APPROVED** [review]
I'm going to go out on a limb and say marketing copy isn't my strong suit but I've left a couple of comments. Very minor just things which struck me as I was reading.

**Comment by dtuite** [comment]
Good suggestions @iain-b . I will improve both of these things before merging.

File: src/components/home/features/ServiceCatalog.js
Comment (punkle) [reply src/components/home/features/ServiceCatalog.js:19]: I wonder should we mention github if it can be used for other repository servers?

**Comment by dtuite** [comment]
@iain-b I have addressed your comments. I'm going to merge this without a +1 since it's just the marketing site and nothing can go too wrong. It's also the weekend and Christmas and I don't want to wait for a review.

**Pull request merged**
